frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.tsx

```diff
@@ -113,7 +113,8 @@ const GitOpsResourcesSection: React.FC<GitOpsResourcesSectionProps> = ({
             </SplitItem>
           </Tooltip>
         )}
-        {degradedResources?.length === 0 && nonSyncedResources.length === 0 && <>&nbsp;</>}
+        {(degradedResources === null || degradedResources.length === 0) &&
```

Comment:
Probably could have just done:
```suggestion
        {(degradedResources?.length === 0) &&
```
But if you're sure you are always `null` and not `undefined` -- it won't matter

Reply:
Ah yeah, the code is either explicitly `null`, an array of data or an empty `[]` 👍